yb-voyager/src/srcdb/source.go

```diff
@@ -18,41 +18,44 @@ package srcdb
-	SSLRootCert           string        `json:"ssl_root_cert"`
-	SSLCRL                string        `json:"ssl_crl"`
-	SSLQueryString        string        `json:"ssl_query_string"`
-	SSLKeyStore           string        `json:"ssl_keystore"`
-	SSLKeyStorePassword   string        `json:"ssl_keystore_password"`
-	SSLTrustStore         string        `json:"ssl_truststore"`
-	SSLTrustStorePassword string        `json:"ssl_truststore_password"`
-	Uri                   string        `json:"uri"`
-	NumConnections        int           `json:"num_connections"`
-	VerboseMode           bool          `json:"verbose_mode"`
-	TableList             string        `json:"table_list"`
-	ExcludeTableList      string        `json:"exclude_table_list"`
-	UseOrafce             utils.BoolStr `json:"use_orafce"`
-	CommentsOnObjects     utils.BoolStr `json:"comments_on_objects"`
-	DBVersion             string        `json:"db_version"`
+	DBType                   string        `json:"db_type"`
+	Host                     string        `json:"host"`
+	Port                     int           `json:"port"`
+	User                     string        `json:"user"`
+	Password                 string        `json:"password"`
+	DBName                   string        `json:"db_name"`
+	CDBName                  string        `json:"cdb_name"`
+	DBSid                    string        `json:"db_sid"`
+	CDBSid                   string        `json:"cdb_sid"`
+	OracleHome               string        `json:"oracle_home"`
+	TNSAlias                 string        `json:"tns_alias"`
+	CDBTNSAlias              string        `json:"cdb_tns_alias"`
+	Schema                   string        `json:"schema"`
+	SSLMode                  string        `json:"ssl_mode"`
+	SSLCertPath              string        `json:"ssl_cert_path"`
+	SSLKey                   string        `json:"ssl_key"`
+	SSLRootCert              string        `json:"ssl_root_cert"`
+	SSLCRL                   string        `json:"ssl_crl"`
+	SSLQueryString           string        `json:"ssl_query_string"`
+	SSLKeyStore              string        `json:"ssl_keystore"`
+	SSLKeyStorePassword      string        `json:"ssl_keystore_password"`
+	SSLTrustStore            string        `json:"ssl_truststore"`
+	SSLTrustStorePassword    string        `json:"ssl_truststore_password"`
+	Uri                      string        `json:"uri"`
+	NumConnections           int           `json:"num_connections"`
+	VerboseMode              bool          `json:"verbose_mode"`
+	TableList                string        `json:"table_list"`
+	ExcludeTableList         string        `json:"exclude_table_list"`
+	UseOrafce                utils.BoolStr `json:"use_orafce"`
+	CommentsOnObjects        utils.BoolStr `json:"comments_on_objects"`
+	DBVersion                string        `json:"db_version"`
+	StrExportObjectTypesList string        `json:"str_export_object_types_list"`
+	ExportObjectTypesList    []string      `json:"export_object_types_list"`
```

Comment:
instead of storing it as a variable  in source struct we can define a method on source for this.
`source.GetExportObjectTypesList()` which will do the same.

I suggest that because it will avoid storing extra and redundant information in the MSR since it is stored and fetched multiple times in a migration.

Reply:
Skipped `ExportObjectTypesList` while creating the MSR json